libs/ui/src/modals/screens/swap-confirmation/swap-confirmation.tsx

```diff
@@ -0,0 +1,40 @@
+import { TransactionResponse } from '@ethersproject/abstract-provider';
+import { RouteProp, useRoute } from '@react-navigation/native';
+import { isDefined, OnEventFn } from '@rnw-community/shared';
+import React, { FC } from 'react';
+
+import { ONE_INCH_SITE, ONE_INCH_ICON } from '../../../api/constants/1inch-agregator';
+import { ScreensEnum, ScreensParamList } from '../../../enums/sreens.enum';
+import { useShelter } from '../../../hooks/use-shelter.hook';
+import { EvmConfirmation } from '../../../screens/send-confirmation/components/evm-confirmation/evm-confirmation';
+import {
+  useSelectedAccountPublicKeyHashSelector,
+  useSelectedNetworkSelector
+} from '../../../store/wallet/wallet.selectors';
+import { DAppHeader } from '../d-app-connection-confirmation/d-app-header/d-app-header';
+
+export const SwapConfirmation: FC = () => {
```

Comment:
why do we have separate screen for swap confirmation?   
could you reuse the confirmation that we already have?

Reply:
We need to show 1icnh logo and icon, and it's hard to use onConfirm from Send page, and that's why I decided to separate logic